src/plugins/data/public/antlr/opensearch_ppl/grammar/OpenSearchPPLParser.g4

```diff
@@ -696,11 +696,11 @@ stringLiteral
    ;
 
 integerLiteral
-   : (PLUS | MINUS)? INTEGER_LITERAL
+   : INTEGER_LITERAL
```

Comment:
if we don't suggest plus or minus, what would be the issue keeping them?

Reply:
I've removed plus and minus from the ignore token list, since I found that commands such as `fields` need them.